src/main/java/io/github/jsonSnapshot/Config.java

```diff
@@ -0,0 +1,5 @@
+package io.github.jsonSnapshot;
+
+public class Config {
```

Comment:
Change that to and interface with a default method that gets the filePath config. Something like this:

`package io.github.jsonSnapshot;

public class SnapshotConfig {
    default String getFilePath() {
        return "src/test/java/";
    }
}`

Reply:
Maybe it's better to create a DefaultConfig implementation instead of creating a default method!